.github/workflows/pip-compile-dev.yml

```diff
@@ -71,4 +74,9 @@ jobs:
       python-versions: "${{ matrix.python-versions }}"
       reset-branch: "${{ inputs.reset-branch || false }}"
       labels: "${{ inputs.labels || 'no_backport,tooling' }}"
-    secrets: inherit
+    # Pass using inherit, as this seems to be the only possible way to access
+    # secrets defined in an enviornment when using nested workflows.
```

Comment:
That workflow should declare an explicit `secrets:` section. Then, passing secrets will work. Just like `inputs:`.

Reply:
Right, but the secrets are part of the `github-bot` environment, and I don't think `environment` can be set in the calling workflow.